mlperf_logging/compliance_checker/hpc_1.0.0/closed_deepcam.yaml

```diff
@@ -14,16 +14,6 @@
     REQ:   EXACTLY_ONE
     CHECK: " v['value'] > 0"
     
-- KEY:
-    NAME:  number_of_nodes
-    REQ:   EXACTLY_ONE
-    CHECK: " v['value'] > 0"
-
-- KEY:
-    NAME:  accelerators_per_node
-    REQ:   EXACTLY_ONE
-    CHECK: " v['value'] > 0"
```

Comment:
So I am assuming we are also moving the check from deepcam to common, right?

Reply:
Yep. `number_of_nodes` and `accelerators_per_node` need to be checked for all 3 HPC benchmarks.